src/libraries/System.Net.Quic/tests/FunctionalTests/QuicStreamTests.cs

```diff
@@ -769,6 +769,206 @@ public async Task AbortAfterDispose_StreamCreationFlushedByDispose_Success()
                 }
             );
         }
+
+        [Fact]
+        public async Task WaitForWriteCompletionAsync_ClientReadAborted_Throws()
+        {
+            const int ExpectedErrorCode = 0xfffffff;
```

Comment:
We might want to choose a different number, diff between 7x `f` and 8x `f` is too small to be noticeable.

Reply:
Assert.Equal will show decimal representation, so it will be easier to distinguish... but it will be harder to catch in the future if some other test will accidentally use 8x`f`

Reply:
I copied the value from a different test. IMO lets leave it as is for now, and if you want to change all instances of this value then it can be done in a future PR.